core/scrollbar.js

```diff
@@ -992,14 +1062,16 @@ Blockly.Scrollbar.prototype.updateMetrics_ = function() {
 
 /**
  * Set the scrollbar handle's position.
- * @param {number} value The distance from the top/left end of the bar, in CSS
- *    pixels.  It may be larger than the maximum allowable position of the
- *    scrollbar handle.
+ * @param {number} value TBD
```

Comment:
This is the same units as `ScrollbarPair.set`. I'm currently leaning towards saying this is the "content displacement relative to the view", since this value is `viewLeft - contentLeft` (or `viewTop - contentTop`).